include/functions.h

```diff
@@ -79,7 +79,7 @@ void func_80258E14(void);
 f32 func_800E34D8(void);
 f32 player_check_collision_below(f32, s32* colliderID);
 s32 can_trigger_loading_zone(void);
-void func_802667F0(s32, Actor*, f32, f32, f32);
+void func_create_textPopup(s32, Actor*, f32, f32, f32);
```

Comment:
You don't need the "func" prefix. what does this function actually do? the mix of snake and camel case is a bit weird to me

Reply:
Oh my bad, I already renamed the function 👍 , it creates the little Nice, Good, Super text that appears when you land action commands successfully.